webpack.config.js

```diff
@@ -35,7 +35,8 @@ const config = module.exports = {
     ]
   },
   entry: {
-    'metamaps.bundle': './frontend/src/index.js'
+    'metamaps.bundle': './frontend/src/index.js',
+    'homepage': './frontend/src/homepage.js',
```

Comment:
it's probably better to include this code in the main bundle

Let's say we want to use the react package in the homepage code (not unreasonable). Now react will be loaded twice; once in the main bundle and once in this bundle, and that causes big bugs with react.

we could make this into a function, either in a window.Metamaps.Homepage.init() option, or just store it in window.Homepage.init()